src/test/java/LoginTests.java

```diff
@@ -15,7 +15,7 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://apps.testpro.io/";
+        String url = "https://testpro.io";
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221401776-7c51997d-fff5-451d-b09e-137194dc81c1.png)